docs/ig.md

```diff
@@ -222,3 +222,29 @@ List of flags:
   `--auto-mount-filesystems` to ig to automatically mount those filesystems.
 - `--pid=host` runs in the host PID namespace. Optional on Linux. This is necessary on Docker Desktop on Windows because
   /host/proc does not give access to the host processes.
+
+### Using ig in a custom Kubernetes pod
+
+If you want to include `ig` in your own container images, you take example on the following Dockerfile:
+
+```Dockerfile
+# In production, you should use a specific version of ig instead of latest:
+# --build-arg BASE_IMAGE=ghcr.io/inspektor-gadget/ig:v0.18.1
+ARG BASE_IMAGE=ghcr.io/inspektor-gadget/ig:latest
+FROM ${BASE_IMAGE} as ig
+
+# Your own image
```

Comment:
What about mentioning that it should work fine in most cases but that the right libc version should be present for it to work?

Reply:
Ah right, I usually use `go build .` on my machine and it doesn't produce a static binary, but `make ig` does. Sorry for the noise.